[ 2023-06-14 09:29:27,607 ] 187 werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.254.142:5000
[ 2023-06-14 09:29:27,608 ] 187 werkzeug - INFO - [33mPress CTRL+C to quit[0m
[ 2023-06-14 09:29:37,560 ] 1414 app - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 13, in index
    return render_template('index.html')
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[ 2023-06-14 09:29:37,586 ] 187 werkzeug - INFO - 127.0.0.1 - - [14/Jun/2023 09:29:37] "[35m[1mGET / HTTP/1.1[0m" 500 -
[ 2023-06-14 09:29:37,795 ] 187 werkzeug - INFO - 127.0.0.1 - - [14/Jun/2023 09:29:37] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
[ 2023-06-14 09:30:59,914 ] 187 werkzeug - INFO - 127.0.0.1 - - [14/Jun/2023 09:30:59] "[33mGET /predict_data HTTP/1.1[0m" 404 -
[ 2023-06-14 09:31:45,405 ] 187 werkzeug - INFO - 127.0.0.1 - - [14/Jun/2023 09:31:45] "[33mGET /predictData HTTP/1.1[0m" 404 -
[ 2023-06-14 09:36:32,444 ] 1414 app - ERROR - Exception on /predictdata [GET]
Traceback (most recent call last):
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 18, in predict_datapoint
    return render_template('home.html')
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home.html
[ 2023-06-14 09:36:32,452 ] 187 werkzeug - INFO - 127.0.0.1 - - [14/Jun/2023 09:36:32] "[35m[1mGET /predictdata HTTP/1.1[0m" 500 -
[ 2023-06-14 09:36:37,733 ] 1414 app - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 13, in index
    return render_template('index.html')
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "C:\Users\sclau\Documents\00-portfolio\mlops\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[ 2023-06-14 09:36:37,740 ] 187 werkzeug - INFO - 127.0.0.1 - - [14/Jun/2023 09:36:37] "[35m[1mGET / HTTP/1.1[0m" 500 -
